Libraries/Lists/__tests__/VirtualizedList-test.js

```diff
@@ -273,4 +273,149 @@ describe('VirtualizedList', () => {
       }),
     );
   });
+
+  it('OnEndReached should not be called after initial rendering', () => {
+    const ITEM_HEIGHT = 100;
```

Comment:
there is quite a bit of duplication in these tests, can you move the duplicated code into the aforementioned `describe` block / beforeEach?